bin/command.php

```diff
@@ -18,25 +19,35 @@
 class Command {
 
 	/**
-	 * Regenerate all doc pages
+	 * Regenerates all doc pages.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--verbose]
+	 * : If set will list command pages as they are generated.
 	 *
 	 * @subcommand gen-all
 	 */
-	public function gen_all() {
+	public function gen_all( $args, $assoc_args ) {
+		// Warn if not invoked with null WP_CLI_CONFIG_PATH.
+		if ( '/dev/null' !== getenv( 'WP_CLI_CONFIG_PATH' ) ) {
```

Comment:
That won't work on Windows. However, not sure if we need to be able to generate the docs on Windows machines...

Reply:
I did actually consider that also believe it or not - only a warning but it is lazy...